Sprint-3/1-key-implement/2-is-proper-fraction.js

```diff
@@ -8,7 +8,10 @@
 // write one test at a time, and make it pass, build your solution up methodically
 
 function isProperFraction(numerator, denominator) {
-    if (numerator < denominator) return true;
+    if (Math.abs(numerator) < Math.abs(denominator) && numerator !=0 && denominator != 0) return true;
```

Comment:
Careful. the `Math.abs()` method just takes any negative number and turns it into a positive which would break this function in certain scenarios.

For example: `console.log(isProperFraction(-5, 3));` would incorrectly return `false` when it is a proper fraction.

Do you know how you could fix this bug? Feel free to message me if you need a hand